lib/src/gemm_blocked_rvm.c

```diff
@@ -1,7 +1,31 @@
 #include "gemm.h"
+#include <riscv_matrix.h>
 
 #define BLOCK_SIZE 4
 
+static inline void process_block_4x4(const size_t k, const float *A, const size_t lda, const float *B, const size_t ldb, float *C, const size_t ldc) {
```

Comment:
Тут моя небольшая опечатка :)
```suggestion
static inline void process_block_4x4(const size_t m, const float *A, const size_t lda, const float *B, const size_t ldb, float *C, const size_t ldc) {
```